idaes/core/util/misc.py

```diff
@@ -117,114 +117,6 @@ def TagReference(s, description=""):
     return r
 
 
-# Author John Eslick
```

Comment:
I think we should leave this in for now with a deprecation warning. You never know who might be using this.

Edit: It looks like you moved this to the `tags` module - you should add a deprecation warning with a redirect here then.

Reply:
Added deprecation warnings for both the old location and the old API.